scripts/tests/test_create_AR_DeployInst.py

```diff
@@ -281,6 +301,85 @@ def test_deploy_without_share(self):
                                       self.reactor_prototypes, 220)
         assert exp == obs
 
+    def test_deploy_without_share3(self):
+        '''
+        Test when the calculated number is negative'''
+        exp = 0
+        obs = di.deploy_without_share('Type1',
+                                      ['Type1', 'Type3', 'Type2'],
+                                      self.reactor_prototypes,
+                                      -100)
+        assert exp == obs
+
+    def test_redeploy_reactors1(self):
+        '''
+        Test when the calculated number is greater than the previous deployment
+        '''
+        exp = 2
+        deploy_schedule = {'DeployInst': {
+            'prototypes': {
+                'val': ['Type1']},
+            'n_build': {
+                'val': [2]},
+            'build_times': {
+                'val': [0]},
+            'lifetimes': {
+                'val': [3]}
+        }}
+        obs = di.redeploy_reactors(250,
+                                   'Type1',
+                                   self.reactor_prototypes,
+                                   deploy_schedule,
+                                   0)
```

Comment:
```suggestion
        obs = di.redeploy_reactors(250,
                                   'Type1',
                                   self.reactor_prototypes,
                                   self.deploy_schedule,
                                   0)
```
I think this should work, since you have `deploy_schedule` attribute, now.